Userland/Libraries/LibGfx/Point.h

```diff
@@ -246,6 +246,12 @@ class Point {
         return Point<U>(roundf(x()), roundf(y()));
     }
 
+    template<typename U>
+    [[nodiscard]] Point<U> to_floored() const
+    {
+        return Point<U>(floorf(x()), floorf(y()));
+    }
```

Comment:
Usage of `floorf` assumes `U` is `float`; usually `.to_type<int>()` is used instead if all values are positive.

Reply:
I see, in that case i have some questions:
1. Since it seems like the coordinates in the image editor are expressed in the range from [0, 0] to [width,height] (correct me if i'm wrong), to fix the issue i could simply use `to_type<int>()`, right? 
2. In such case, wouldn't that be like reverting the change Andreas made in e6d860f2fe49ad8f39300087d1d9da3e052f3faf? And should i remove the to_floored function then? I assume there may be situations where flooring a point would be needed in the future

Reply:
1: yes
2: yes, and yes: we shouldn't add functions just because they _could_ be used, they should actively be used in the PR

You should doublecheck whether this PR isn't breaking the very thing Andreas has tried to fix, the very least :-)